ksqldb-common/src/main/java/io/confluent/ksql/serde/SerdeOption.java

```diff
@@ -28,7 +28,7 @@
    *
    * @see SerdeOption#UNWRAP_SINGLE_VALUES
    */
-  WRAP_SINGLE_VALUES(SerdeFeature.WRAP_SINGLES),
+  WRAP_SINGLE_VALUES(SerdeFeature.WRAP_SINGLES, false),
```

Comment:
Ignore the changes in this file - this is work I need to do, but was needed to get the examples working.

Reply:
see:  https://github.com/confluentinc/ksql/pull/6256